extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -0,0 +1,642 @@
+		flexContainer.addItem(titleContainer, titleFlex);
+
+		let configurationSection = this.modelView.modelBuilder.divContainer().component();
+		configurationSection.addItem(flexContainer);
+
+		return configurationSection;
+	}
+
+	private editVCores(type: string) {
+		let currentCPUSize: string | undefined;
+		if (type === loc.vCoresMax) {
+			currentCPUSize = this._postgresModel.config?.spec.scheduling?.default?.resources?.requests?.cpu;
+
+			if (!currentCPUSize) {
+				currentCPUSize = '0';
+			}
+
+			this.vCoresMaxBox!.value = currentCPUSize;
+
+			this.disposables.push(
+				this.vCoresMaxBox!.onTextChanged(() => {
+					if (!(this.handleOnTextChanged(this.vCoresMaxBox!, currentCPUSize!))) {
+						this.saveArgs!.coresRequest = undefined;
+					} else {
+						this.saveArgs!.coresRequest = this.vCoresMaxBox!.value;
+					}
+				})
+			);
+		} else if (type === loc.vCoresMin) {
+			currentCPUSize = this._postgresModel.config?.spec.scheduling?.default?.resources?.limits?.cpu;
+
+			if (!currentCPUSize) {
+				currentCPUSize = '0';
+			}
+
+			this.vCoresMinBox!.value = currentCPUSize;
+
+			this.disposables.push(
+				this.vCoresMinBox!.onTextChanged(() => {
+					if (!(this.handleOnTextChanged(this.vCoresMinBox!, currentCPUSize!))) {
+						this.saveArgs!.coresLimit = undefined;
+					} else {
+						this.saveArgs!.coresLimit = this.vCoresMinBox!.value;
+					}
+				})
+			);
+		}
+	}
+
+	private editMemory(type: string) {
+		let currentMemorySize: string | undefined;
+		let currentMemSizeConversion: string;
+
+		if (type === loc.memoryMax) {
+			currentMemorySize = this._postgresModel.config?.spec.scheduling?.default?.resources?.requests?.memory;
+
+			if (!currentMemorySize) {
+				currentMemSizeConversion = '0';
+			} else {
+				currentMemSizeConversion = this.gibibyteConversion(currentMemorySize);
+			}
+
+			this.memoryMaxBox!.value = currentMemSizeConversion!;
+
+			this.disposables.push(
+				this.memoryMaxBox!.onTextChanged(() => {
+					if (!(this.handleOnTextChanged(this.memoryMaxBox!, currentMemSizeConversion!))) {
+						this.saveArgs!.memoryRequest = undefined;
+					} else {
+						this.saveArgs!.memoryRequest = this.memoryMaxBox!.value + 'Gi';
+					}
+				})
+			);
+		} else if (type === loc.memoryMin) {
+			currentMemorySize = this._postgresModel.config?.spec.scheduling?.default?.resources?.limits?.memory;
+
+			if (!currentMemorySize) {
+				currentMemSizeConversion = '0';
+			} else {
+				currentMemSizeConversion = this.gibibyteConversion(currentMemorySize);
+			}
+
+			this.memoryMinBox!.value = currentMemSizeConversion!;
+
+			this.disposables.push(
+				this.memoryMinBox!.onTextChanged(() => {
+					if (!(this.handleOnTextChanged(this.memoryMinBox!, currentMemSizeConversion!))) {
+						this.saveArgs!.memoryLimit = undefined;
+					} else {
+						this.saveArgs!.memoryLimit = this.memoryMinBox!.value + 'Gi';
+					}
+				})
+			);
+		}
+	}
+
+	private gibibyteConversion(value: string): string {
```

Comment:
`gigibyteConversion` - although I think convertToGigibyteString or something is even more clear